### PR TITLE
MENFORCER-435: Fixing unit tests

### PR DESCRIPTION
**Pull request by ajarmoniuk**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [X] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MENFORCER) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [X] Each commit in the pull request should have a meaningful subject line and body.
 - [X] Format the pull request title like `[MENFORCER-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MENFORCER-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [X] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [X] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [X] You have run the integration tests successfully (`mvn -Prun-its clean verify`).

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [X] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [X] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by ajarmoniuk** [comment]
@slawekjaranowski fixes the unit test problems you spotted.

File: enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/BannedDependenciesBase.java [hunk 1]
```diff
@@ -20,12 +20,12 @@
 
 import java.util.List;
 
-import org.apache.commons.lang3.StringUtils;
 import org.apache.maven.artifact.Artifact;
 import org.apache.maven.enforcer.rule.api.EnforcerRuleException;
 import org.apache.maven.enforcer.rule.api.EnforcerRuleHelper;
 import org.apache.maven.execution.MavenSession;
 import org.apache.maven.plugins.enforcer.utils.ArtifactUtils;
+import org.apache.maven.shared.utils.StringUtils;
```
Comment (slawekjaranowski) [inline enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/BannedDependenciesBase.java:28]: Why change commons-lang to Maven shared?
Even more I would to like remove Maven shared from dependencies in the future.

Comment (ajarmoniuk) [reply enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/BannedDependenciesBase.java]: Even worse. The reason was my IDE changing it to Google something during a failed rebase, which was the reason for the red build. Corrected.

**Comment by jira-importer** [comment]
Resolve #556

**Comment by jira-importer** [comment]
Resolve #556

**Pull request merged**